### PR TITLE
samples(): removed unused dependencies

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Removed unused dependencies ( dependencies dev `@types/ws`), in the examples:
- [`04-grpc`](https://github.com/nestjs/nest/tree/master/sample/04-grpc)
- [`07-sequelize`](https://github.com/nestjs/nest/tree/master/sample/07-sequelize)

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
This PR, Removed `@types/ws` dependencies dev as the use of websockets is not used, in the examples:
- [`04-grpc`](https://github.com/nestjs/nest/tree/master/sample/04-grpc)
- [`07-sequelize`](https://github.com/nestjs/nest/tree/master/sample/07-sequelize)

### CODE REVIEW

File: sample/07-sequelize/package.json [hunk 1]
```diff
@@ -33,26 +33,25 @@
   },
   "devDependencies": {
     "@nestjs/cli": "8.1.5",
-    "@types/sequelize": "4.28.10",
     "@nestjs/schematics": "8.0.5",
     "@nestjs/testing": "8.2.3",
     "@types/express": "4.17.13",
     "@types/jest": "27.0.3",
     "@types/node": "16.11.17",
+    "@types/sequelize": "4.28.10",
```
Comment (WikiRik) [inline sample/07-sequelize/package.json:41]: Sequelize comes with its own types so this is not needed

Comment (Tony133) [reply sample/07-sequelize/package.json]: Removed, if I'm not mistaken from sequelize v6 you don't need this `@types/sequelize` package right? only version 5 and earlier needed this.
 
Thanks for the reporting 👍

Comment (WikiRik) [reply sample/07-sequelize/package.json]: Sequelize v5 also ships with its own tests, I'm not sure how complete those are though

Comment (Tony133) [reply sample/07-sequelize/package.json]: However, Sequelize v6 no more types are needed via package(`@types/sequelize`), as they are built in, right?

**Comment by Tony133** [comment]
Conflits resolved

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**